src/registrar/tests/test_admin.py

```diff
@@ -421,29 +422,27 @@ def test_short_org_name_in_applications_list(self):
         # Now let's make sure the long description does not exist
         self.assertNotContains(response, "Federal: an agency of the U.S. government")
 
-    @boto3_mocking.patching
-    def test_save_model_sends_submitted_email(self):
-        # make sure there is no user with this email
-        EMAIL = "mayor@igorville.gov"
-        User.objects.filter(email=EMAIL).delete()
+    def transition_state_and_send_email(self, application, status):
```

Comment:
I'm 99% sure this is fine, but Boto is kind of temperamental, so I would just make sure that this method isn't firing out an email (you can add a log event to the `_send_status_update_email(` function I think)

Reply:
Put a logger in there, didn't seem to trigger